testexec/testexec.go

```diff
@@ -21,6 +24,8 @@ import (
 	"github.com/warpfork/go-testmark"
 )
 
+var DisableStrict = flag.Bool("testexec.no-strict", false, "Setting this flag will allow unconventional directory structures.")
```

Comment:
Is this desirable?  When would someone want to do this for one run, and not commit that preference in the source control?

I'm not opposed it it either if you feel strongly about it because it's not really adding much maintenance burden.

(I may have accidentally "asked" for this by saying "flag" when we talked about this in chat -- if so, I'm sorry for that; I often say 'flag" when I mean "a boolean", and didn't necessarily mean to express a desire to expose it as a CLI flag.)

Reply:
Yeah I'll pull this.